HW03 - Carpet Installation/CarpetInstallation.c

```diff
@@ -30,28 +30,25 @@ int main() {
     //Prints the charges section.
     printCharges(length, width, costSF, installPrice, discount, subtotal, total);
 
-    return(0);
+    return 0;
 }
 
 //Gets carpet dimensions from the user
 void getInput(int* length, int* width, int* discount, float* costSF) {
-    printf("\nEnter the length of the room: ");
+    printf("\nEnter the length of the room (in ft): ");
     scanf("%d", length);
-    printf("Enter the width of the room: ");
+    printf("Enter the width of the room (in ft): ");
     scanf("%d", width);
-    printf("Enter the customer discount: ");
+    printf("Enter the customer discount (a percent value from 0 to 100): ");
     scanf("%d", discount);
     printf("Enter the cost per square foot: ");
     scanf("%f", costSF);
-    
-    return;
 }
 
 //Calculates price of carpet installation
 float calcInstalledPrice(int length, int width, float costSF) {
     float installPrice = 0;  //Price of installation.
-    installPrice = (length * width) * costSF;
-    installPrice += (length * width) * 0.35;
+    installPrice = ((length * width) * costSF)+ ((length * width)*0.35);
```

Comment:
Wonky spacing between the operators. Also while it was that way originally, there is no need for a set of parenthesis around `length` and `width` as each item in the group is just being multiplied together.